### PR TITLE
add solution for day1 exercise .

### PR DESCRIPTION
**Pull request by ankitmahajan1**

### CODE REVIEW

File: day1-exercie.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+<details>
+
+<summary></summary>
+
+`kubectl create namespace secret`
+
+</details>
+
+
+2. Get all the namespaces in k8s
```
Comment (NitishAroraKPMG) [inline day1-exercie.md:16]: Move this section/question to the top

File: day1-exercie.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+<details>
+
+<summary></summary>
+
+`kubectl get namespaces`
+
+</details>
+
+
+3. Use describe command for the namespace created
```
Comment (NitishAroraKPMG) [inline day1-exercie.md:27]: Let us ask the question, not the answer in the question itself, like:-

Get the details of the namespace secret.

File: day1-exercie.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+<details>
+
+<summary></summary>
+
+    `kubectl describe namespace secret`
+
+</details>
+
+
+4. Create a pod names nginx-demo with image nginx with tag 1.42
```
Comment (NitishAroraKPMG) [inline day1-exercie.md:38]: Create a pod named nginx-demo with image nginx, using the tag 1.42

File: day1-exercie.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+<details>
+
+<summary></summary>
+
+`kubectl delete pod nginx-demo`
+
+</details>
+
+
+7. Create a new pod with name nginx-secret under namespace secret (created in step 1)
```
Comment (NitishAroraKPMG) [inline day1-exercie.md:77]: Create a pod named nginx-secret in secret namespace.

File: day1-exercie.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+<details>
+
+<summary></summary>
+
+`kubectl run nginx-secret --image nginx:1.22-bullseye -n secret`
+
+</details>
+
+
+8. Check the status of the pod created
```
Comment (NitishAroraKPMG) [inline day1-exercie.md:89]: Check the status of nginx-secret pod.

File: day1-exercie.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+<summary></summary>
+
+`kubectl get pods nginx-secret -n secret` (to get particular named pod created under namespace)
+
+`kubectl get pods -n secret` (to get all pods for namespace secret)
+
+</details>
+
+
+9. Delete the namespace secret (use command kubectl delete)
```
Comment (NitishAroraKPMG) [inline day1-exercie.md:102]: Don't specify anything in the question itself.

File: day1-exercie.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+<details>
+
+<summary></summary>
+
+`kubectl delete ns secret`
+
+</details>
+
+
+10. Try to get the pod under namespace secret
```
Comment (NitishAroraKPMG) [inline day1-exercie.md:113]: Get the pod(s) under the secret namespace.

File: day1-exercie.md (hunk 1 above)
Comment (ankitmahajan1) [reply day1-exercie.md:16]: done

File: day1-exercie.md (hunk 2 above)
Comment (ankitmahajan1) [reply day1-exercie.md:27]: make sense

File: day1-exercie.md (hunk 3 above)
Comment (ankitmahajan1) [reply day1-exercie.md:38]: done

File: day1-exercie.md (hunk 4 above)
Comment (ankitmahajan1) [reply day1-exercie.md:77]: done

File: day1-exercie.md (hunk 5 above)
Comment (ankitmahajan1) [reply day1-exercie.md:89]: ok

File: day1-exercie.md (hunk 6 above)
Comment (ankitmahajan1) [reply day1-exercie.md:102]: ok

File: day1-exercie.md (hunk 7 above)
Comment (ankitmahajan1) [reply day1-exercie.md:113]: done

File: day1-exercie.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+<details>
+
+<summary></summary>
+
+`kubectl create namespace secret`
+
+</details>
+
+
+3. Get detials of the namespace created
```
Comment (NitishAroraKPMG) [inline day1-exercie.md:27]: typo - details

Get details of the secret namespace.

File: day1-exercie.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+<details>
+
+<summary></summary>
+
+`kubectl delete pod nginx-demo`
+
+</details>
+
+
+7. Create a pod named nginx-secret in secret namespace.
```
Comment (NitishAroraKPMG) [inline day1-exercie.md:77]: Try to put as a one-liner.

Create a pod named nginx-secret, using the image nginx with tag 1.22-bullseye in secret namespace.

**Review by NitishAroraKPMG — COMMENTED** [review]
correct the file name ... there is a typo in the file name also

File: day1-exercie.md (hunk 9 above)
Comment (ankitmahajan1) [reply day1-exercie.md]: done

**Review by NitishAroraKPMG — APPROVED** [review]

**Pull request merged**